### PR TITLE
PAR-1197  minor config update

### PR DESCRIPTION
**Pull request by jon-edery**

### CODE REVIEW

**Comment by jon-edery** [comment]
Good idea have created a new filter that uses the internal status config rather then hard coded values as it was built previously.

File: sync/views.view.par_user_partnerships.yml [hunk 1]
```diff
@@ -681,7 +678,7 @@ display:
                 operator: '='
                 value: confirmed_authority
               3:
-                title: 'Confirmed by Business'
+                title: 'Confirmed by Organisation'
```
Comment (kalpaitch) [inline sync/views.view.par_user_partnerships.yml]: We should probably get this filter off the 'default' view config but given that it's not used it's hardly important right now.

File: sync/views.view.par_user_partnerships.yml [hunk 2]
```diff
@@ -1362,6 +1359,343 @@ display:
       display_extenders: {  }
       path: partnerships
       display_description: ''
+      filters:
```
Comment (kalpaitch) [inline sync/views.view.par_user_partnerships.yml]: Can't see the existing filter as having been removed, will import and check in a mo.

**Comment by stemount** [comment]
When all traces of the old filter gone then please feel free to dismiss my Request changes review 💯

**Pull request merged**